test/internet/test-tls-reuse-host-from-socket.js

```diff
@@ -1,27 +1,17 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
 
 if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-var tls = require('tls');
+const tls = require('tls');
 
-var net = require('net');
-var connected = false;
-var secure = false;
+const net = require('net');
 
-process.on('exit', function() {
-  assert(connected);
-  assert(secure);
-  console.log('ok');
-});
-
-var socket = net.connect(443, 'www.google.com', function() {
-  connected = true;
-  var secureSocket = tls.connect({ socket: socket }, function() {
-    secure = true;
+const socket = net.connect(443, 'www.google.com', common.mustCall(() => {
+  const secureSocket = tls.connect({socket: socket}, common.mustCall(() => {
     secureSocket.destroy();
-  });
-});
+    console.log('ok');
```

Comment:
I left it in to preserve the original test's logging. But I'm happy to take it out.